vta/hardware/intel/scripts/soc_system.tcl

```diff
@@ -0,0 +1,134 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+package require -exact qsys 16.0
+
+create_system soc_system
+
+set_project_property DEVICE_FAMILY {Cyclone V}
+set_project_property DEVICE {5CSEBA6U23I7}
```

Comment:
can we pass DEVICE as an argument here:  `set DEVICE [lindex $argv 0]`

Reply:
Both DEVICE and DEVICE_FAMILY has been defined in the [pkg_config.py](https://github.com/dmlc/tvm/pull/3494/files#diff-842ae8b85a7944f55efd432de2d10994), parameterized in [Makefile](https://github.com/dmlc/tvm/pull/3494/files#diff-81af3620e4013b66642df37ed1408010) and [soc_system.tcl](https://github.com/dmlc/tvm/pull/3494/files#diff-b87eb483891e96c914c4464557ba1871).